text/0000-add-scratch-stack.md

```diff
@@ -0,0 +1,161 @@
+# Create scratch stack
+
+## Summary
+
+The Paketo stacks maintainers should create and maintain a scratch stack that
+can run statically-linked applications (e.g. golang). The Builder team does not
```

Comment:
```suggestion
can run statically-linked applications (e.g. golang, Rust). The Builder team does not
```

Reply:
Happy to add Rust in the summary. Initially I omitted it from the summary because it's not a community-supported language whereas golang is a fully-supported buildpack.